doc/conf.py

```diff
@@ -345,7 +345,8 @@ def setup(app):
     'subsection_order': ExplicitOrder(['../examples/sin_func',
                                        '../examples/no_output',
                                        '../tutorials/seaborn']),
```

Comment:
two of these folders don't exist anymore right?

Reply:
Good point